src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp

```diff
@@ -382,7 +382,13 @@ int os::extra_bang_size_in_bytes() {
 
 extern "C" {
   int SpinPause() {
-    return 0;
+    using spin_wait_func_ptr_t = void (*)();
```

Comment:
I think you'll want `ThreadWXEnable wx(WXExec, thread);` for Apple here.

Reply:
Sorry, why do we need it? This is linux-aarch64 implementation of SpinPause.
I think it should in SpinPause in `os_cpu/bsd_aarch64/os_bsd_aarch64.cpp`. Am I right?

Reply:
You are indeed right. We can worry about Apple another day.